cookbook/doctrine/registration_form.rst

```diff
@@ -64,6 +87,16 @@ You have a simple ``User`` entity mapped to the database::
             $this->email = $email;
         }
 
+        public function getUsername()
+        {
+            return $this->username;
+        }
+
+        public function setUsername($username)
```

Comment:
Ah no, we need to keep it. Doctrine doesn't need the setter, but the Form component does.